inception/inception-log/src/main/java/de/tudarmstadt/ukp/inception/log/EventLoggingListener.java

```diff
@@ -86,7 +86,7 @@ public void onApplicationEvent(ApplicationEvent aEvent)
 
         var adapter = maybeAdapter.get();
 
-        if (properties.getExcludeEvents().contains(adapter.getEvent(aEvent))) {
+        if (!properties.shouldLogEvent(adapter.getEvent(aEvent))) {
```

Comment:
IMHO `EventLoggingProperties` should remain a data class. That is I would tend to move "shouldLogEvent" into the logging service instead - maybe as a static / package-private method so it can be tested without having to bring up a Spring Boot context.